smart-cards/src/main/java/gov/va/api/health/smartcards/patient/ImmunizationTransformer.java

```diff
@@ -16,18 +21,44 @@ static Reference referenceOnly(@NonNull Reference reference) {
     return Reference.builder().reference(reference.reference()).build();
   }
 
-  Reference location() {
-    if (entry.resource().location() == null) {
-      return null;
+  String occurrenceDateTime() {
+    // only publish the date information. time is not necessary.
+    String odt = entry.resource().occurrenceDateTime();
+    // expect DQ to provide a ISO8601-formatted date
+    if (odt.contains("T")) {
+      odt = Iterables.get(Splitter.on('T').split(odt), 0);
     }
-    return referenceOnly(entry.resource().location());
+    return odt;
   }
 
   Reference patient() {
     // Do not include display
     return referenceOnly(entry.resource().patient());
   }
 
+  List<Immunization.Performer> performer() {
+    if (CollectionUtils.isEmpty(entry.resource().contained())) {
```

Comment:
DQ doesn't model contained resources, so this will always be null. I think `entry.resource().location()` is what you want to check?

Reply:
Ok, I see now that these are populated before the transformer is invoked

Reply:
Yeah- it's not immediately obvious, but the Controller injects the Location resource before running this transformer.